data/ui/window.ui

```diff
@@ -159,7 +159,7 @@
                             <property name="margin-top">30</property>
                             <property name="margin-start">30</property>
                             <property name="margin-bottom">20</property>
-                            <property name="label" translatable="true">Palettes</property>
+                            <property name="label" translatable="yes">Palettes</property>
```

Comment:
This should be "true" not "yes". "yes" isn't a thing. :P

Reply:
Other lines are `translatable="yes"` tagged 🙃

Reply:
Oh.